src/interfaces/ecpg/test/preproc/meson.build

```diff
@@ -0,0 +1,38 @@
+
+pgc_files = [
+  'array_of_struct',
+  'autoprep',
+  'comment',
+  'cursor',
+  'define',
+  'init',
+  'outofscope',
+  'pointer_to_struct',
+  'strings',
+  'type',
+  'variable',
+  'whenever',
+  'whenever_do_continue',
+]
+
+pgc_extra_flags = {
+  'array_of_struct': ['-c'],
+  'pointer_to_struct' : ['-c'],
+  'autoprep': ['-r', 'prepare'],
+  'strings': ['-i'],
+}
+
+foreach pgc_file: pgc_files
+  exe_input = custom_target(pgc_file,
+    input: '@0@.pgc'.format(pgc_file),
```

Comment:
Seems like we could just use have pgc_files be the actual filenames, instead of the .format() thing?

Reply:
If I apply that change,
```
executable(pgc_file,
...
)
```

 will be a problem.